models/paraswap/zkevm/paraswap_v6_zkevm_trades.sql

```diff
@@ -0,0 +1,135 @@
+{{ config(
+    schema = 'paraswap_v6_zkevm',
+    alias = 'trades',    
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'method', 'trace_address'],
+    post_hook='{{ expose_spells(\'["zkevm"]\',
+                                "project",
+                                "paraswap_v6",
+                                \'["eptighte", "mwamedacen"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2024-03-01' %}
+
+with dexs AS (
+    SELECT 
+        blockTime AS block_time,
+        blockNumber AS block_number,
+        from_hex(beneficiary) AS taker, 
+        null AS maker,  -- TODO: can parse from traces
+        receivedAmount AS token_bought_amount_raw,
+        fromAmount AS token_sold_amount_raw,
+        CAST(NULL AS double) AS amount_usd,  
+        method,              
+        CASE 
+            WHEN from_hex(destToken) = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+            THEN 0x4f9a0e7fd2bf6067db6994cf12e4495df938e6e9 -- WETH 
+            ELSE from_hex(destToken)
+        END AS token_bought_address,        
+        CASE 
+            WHEN from_hex(srcToken) = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+            THEN 0x4f9a0e7fd2bf6067db6994cf12e4495df938e6e9 -- WETH 
+            ELSE from_hex(srcToken)
+        END AS token_sold_address,
+        projectContractAddress as project_contract_address,
+        txHash AS tx_hash, 
+        callTraceAddress AS trace_address,
+        CAST(-1 as integer) AS evt_index
+    FROM {{ ref('paraswap_v6_zkevm_trades_decoded') }}     
+    {% if is_incremental() %}
+    WHERE blockTime >= date_trunc('day', now() - interval '7' day)
```

Comment:
We can use the `incremental_predicate` here and for all incremental filters:
```suggestion
    WHERE {{ incremental_predicate('blockTime') }}
```